pyxrf/model/draw_image.py

```diff
@@ -552,6 +558,10 @@ def _adjust_data_range__min_ratio(c_min, c_max, c_axis_range, *, min_ratio=0.01)
             else:
 
                 maxz = np.max(data_dict)
+                # Set some reasonable minimum range for the colorbar
+                #   Zeros or negative numbers will be shown in white
+                if maxz <= 1e-30:
```

Comment:
Thanks for the fix. That was actually my question.

Reply:
Yes, I noticed it by accident. Currently the colorbar axis labels are not displayed for log plots, so I didn't notice it. We are waiting for Matplotlib 3.2.

Reply:
OK.